antiplag/models.py

```diff
@@ -17,6 +19,24 @@ class SubmissionStatus(models.TextChoices):
     def __str__(self):
         return f"Submission {self.id} ({self.status})"
 
+    @classmethod
+    def count(cls):
+        cached_value = cache.get("submission_count", None)
+        return cached_value if cached_value else cls.objects.count()
+
+    @classmethod
+    def average_time(cls):
+        cached_value = cache.get("submission_avg_time")
```

Comment:
Why are you using `.get()` without the fallback argument? Method higher it is used with the fallback argument.

`cache.get(key)` returns `None` as default, if no cached item is found, but I think specifying `None` makes it a bit more readable.

Reply:
Yes I know, previously I had it implicitly, but I like it explicit - therefore I removed checks for `X is not None` and replaced with `not X`, but I forgot to replace this part as well. 

Though, this discussion is irrelevant, I'm going to refactor it to view caching as you suggested.